scripts/webhook-deploy.sh

```diff
@@ -57,6 +57,8 @@ done
 [ -z ${mode} ] && mode="preprogrammed"
 [ -z ${namespace} ] && namespace="default"
 
+which ${kubectl} > /dev/null 2>&1 || (echo "ERROR: ${kubectl} not found"; exit 1)
```

Comment:
This happens to exit the whole script as intended if script is executed like `path/to/webhook-deploy.sh`, because shebang launches bash with `-e`. Without `-e` executing the script continues after printing the error like in webhook-create-signed-cert.sh. So I'd recommend statement block `{ ... }` here, too.

Reply:
Thanks! I've updated two more place with problematic code.